src/main/java6/net/finmath/time/ScheduleGenerator.java

```diff
@@ -127,8 +126,8 @@ private ScheduleGenerator() {
 	 * t = 0 corresponds to the reference date.
 	 * 
 	 * @param referenceDate The date which is used in the schedule to internally convert dates to doubles, i.e., the date where t=0.
-	 * @param startDate The start date of the first period.
-	 * @param maturity The end date of the first period.
+	 * @param startDate The start date of the first period (this may/should be an unadjusted date).
```

Comment:
I copied the @param for startDate and maturityDate from the upper most createScheduleFromConventions() method - which is effectively called by every other createScheduleFromConventions() methods. Giving it a 2nd thought: The maturityDate definitively has to be unadjusted, imO the startDate will always be a busDay (so adjusted=unadjusted and the comment is redundant).